src/main/java/com/conveyal/gtfs/GTFSCache.java

```diff
@@ -96,14 +74,18 @@ private void createCache(int cacheSize) {
             }
         };
         this.cache = CacheBuilder.newBuilder()
-            .maximumSize(cacheSize)
-            .removalListener(removalListener)
-            .build(new CacheLoader<String, GTFSFeed>() {
-                @Override
-                public GTFSFeed load(String s) throws Exception {
-                    return retrieveFeed(s);
-                }
-            });
+                .maximumSize(cacheSize)
+                .removalListener(removalListener)
+                .build(new CacheLoader<String, GTFSFeed>() {
+                    @Override
+                    public GTFSFeed load(String s) throws Exception {
+                        return retrieveFeed(s);
+                    }
+                });
+    }
+
+    public long getCurrentCacheSize() {
+        return this.cache.size();
```

Comment:
No, I meant for it to return the current size (in case it's needed at some point?).  Max size was meant to be grabbed from `GTFSCache.cacheSize`.